### DISCUSSION TITLE
buddhism/2023/leaks/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#40] [General]
# buddhism/leaks/

https://shanenull.com/buddhism/2023/leaks/

**Comment by shane0** [comment]
A leak, or asrava, in Buddhism refers to the mental and physical outflows that lead to suffering and dissatisfaction. There are three types of leaks:

* **Sensual leak (kama-asrava)**: This is the attachment to sensual pleasures, such as sights, sounds, tastes, smells, and touch.
* **Existence leak (bhava-asrava)**: This is the craving for existence, including birth, death, and rebirth.
* **Ignorance leak (avijja-asrava)**: This is the root of all suffering, and it includes the lack of understanding of the true nature of reality.

The three leaks are interconnected, and they all contribute to the cycle of suffering. The sensual leak leads to the existence leak, as we crave for more and more sensual pleasures. The existence leak leads to the ignorance leak, as we become attached to our existence and fail to see the true nature of reality.

The goal of Buddhism is to end the cycle of suffering by stopping the three leaks. This is done through meditation and practice of the Eightfold Path.

Here are some examples of how the three leaks can manifest in our lives:

* **Sensual leak:** A person who is attached to sensual pleasures may spend a lot of time and energy pursuing them, even if it leads to negative consequences, such as addiction or financial problems.
* **Existence leak:** A person who is attached to existence may be afraid of death or change. They may also be constantly striving for more, even if they already have everything they need.
* **Ignorance leak:** A person who is ignorant of the true nature of reality may believe that they are separate from others and the world around them. They may also have false beliefs about what is important in life.

By practicing Buddhism, we can learn to let go of the three leaks and achieve true happiness and freedom from suffering.

**Comment by shane0** [comment]
ideos about Buddhism Asrava:

* [Bhante Punnaji explains the meaning of Āsava (*sanskrit: Āśrava)](http://www.youtube.com/watch?v=SL02pkFQAgA) by Bhante Punnaji
* [Jainism Explained](http://www.youtube.com/watch?v=KkwmYlgkOhU) by Cogito
* [15 - Aashrav Bhavanaa Aaswaad](http://www.youtube.com/watch?v=4KgwRzsnWX4) by Yesha Shah
* [आस्रव कितने है ?  बुद्ध तथा उनके सन्देश #shorts   #buddha   #बुद्ध](http://www.youtube.com/watch?v=0m1GWxLtAJU) by Truth of Life
* [Jainism Nav-Tattvas (Jiva, Punya, Papa, Asrava, Samvara, Nirjara, Moksha, Bandha) In Hindi](http://www.youtube.com/watch?v=u6LMyah1MDA) by Harshit Dwivedi Education
http://googleusercontent.com/youtube_content/10

**Comment by shane0** [comment]
create print publish cards

**Comment by shane0** [comment]
"A further teaching on Right View is that when a noble disciple understands the leaks,... he or she has Right View. There are three leaks: the leak of sensual desire, the leak of being, and the leak of ignorance. The three leaks originate where ignorance originates.

Maj jhima Nikaya 9

**Comment by shane0** [comment]
https://www.dhammatalks.org/suttas/SN/SN3_17.html

**Comment by shane0** [comment]
https://www.dhammatalks.org/suttas/MN/MN9.html

**Comment by shane0** [comment]
bhante translates as: breaking the influences tainted unstable externals flowing through 

his fix is the supernormal 8fold path

**Comment by shane0** [comment]
[diamond sutra](https://shanenull.com/buddhism/diamond_sutra/)
